sunscreen_math/src/lib.rs

```diff
@@ -36,6 +36,9 @@ pub mod stats;
 /// Functions related to calculating security and correctness parameters.
 pub mod security;
 
+/// Functions and data structures related to geometry
```

Comment:
would it be helpful for future reference to include in comment the name of the functions/DS that necessitated this addition?

Reply:
Indeed!